cmd/Dockerfile

```diff
@@ -0,0 +1 @@
+# Multi-stage dockerfile building a container image with both binaries included
```

Comment:
Should the binaries be built from within a builder image?

Reply:
You can do multistage build where you have both the builder and production images in the same Dockerfile and you can copy from one to another.